driver-3/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 ## Release History
 
+## 1.0.1
+
+This is a maintenance release that addresses these `CosmosLoadBalancingPolicy` issues:
+
+- Preferred regions are now properly ordered when the primary region is explicitly specified.
+  Prior to this release, if the primary region was explicitly specified in the list of preferred regions, it would be 
+  moved to the end of the preferred region list.
+
+- `CosmosLoadBalancingPolicy::onDown` now removes hosts based on endpoint address, not datacenter name.
+   This behavior change avoids a problem that arises when a host is removed before its datacenter name has been
+   determined. It also ensures that the address of the host removed matches the address of the host to be
+   removed.
+
+- Writes now fail over to secondary regions when the primary region is down and multi-region-writes are disabled.
```

Comment:
Is it fair to say that the the behavior for multi-region accounts is the same now, whether multi-region writes are turned on or not?

Reply:
Not quite. Writes alway go to the primary first.